build_scripts/build_usd.py

```diff
@@ -910,6 +1017,17 @@ def InstallBoost(context, force, buildArgs):
                    "include/boost-1_70/boost/version.hpp",
                    "include/boost-1_78/boost/version.hpp")
 
+############################################################
+# IOS toolchain
+IOS_TOOLCHAIN_URL = "https://github.com/leetal/ios-cmake/archive/refs/tags/4.3.0.tar.gz"
```

Comment:
I still prefer that we create our own toolchain file, or, that we intern a copy of Bogdan's cmake file here. The license on the file permits interning the file, and has no copyleft complications.

Reply:
Agree! I'll modify it to intern based on the license being suitable for that

Reply:
I see the new commit :)